registration/js/registration.js

```diff
@@ -851,6 +856,12 @@ SeatReg.prototype.generateField = function(fieldName, isLastCheckItem) {
 	return label;
 };
 
+SeatReg.prototype.generateCustomFooterFextField = function (customFooterText) {
+	var decodedcustomFooterText = $("<div/>").html(customFooterText).text();
+	var div = $('<div class="field-label custom-footer-text">'+ decodedcustomFooterText + '</div>');
+	return div;
```

Comment:
Lets remove script tags before returning the `div`

`div.find('script').remove();`

Reply:
Changed: https://github.com/SiimKirjanen/seatreg/commit/f4e645f5ff3754d07f0991e102f06bd28c8651d9